src/app/public/modules/type-definitions/anchor-link.service.ts

```diff
@@ -28,7 +30,11 @@ export class SkyDocsAnchorLinkService {
     this.anchorIds = typeDefinitionsProvider.anchorIds;
   }
 
-  public applyTypeAnchorLinks(content: string): string {
+  /**
+   * Formats known type names with `<code>` tags and wraps them with anchor tags, linking to the appropriate type.
+   * If the content is already contained within a `<code>` tag, set `codeFormat = false` to prevent extra `<code>` tags from being added.
+   */
+  public applyTypeAnchorLinks(content: string, codeFormat: boolean = true): string {
```

Comment:
What do you think about making the second parameter an object so that we can add more options in the future? Also, you used `addCodeFormatting` as the parameter name in the pipe, which I feel works better. Maybe even `applyCodeFormatting`?
```
{
  applyCodeFormatting: boolean;
}
```

Reply:
Yeah, I'm good with that. I'll make the change.

Reply:
Done.